renderer/components/exports/export.js

```diff
@@ -75,7 +80,9 @@ export default class Export extends React.Component {
           </div>
         </div>
         <div className="details">
-          <div className="title">{defaultFileName}</div>
+          <div className={'title ' + (status === 'processing' ? 'title-color-disabled' : 'title-color')}>
```

Comment:
Can we use classnames for this?

Reply:
Sorry, not sure I follow. Do you mean remove `title` class and duplicate its properties in `title-color` and `title-color-disabled`?

Reply:
Oh, you mean using `classnames` attribute instead of `classname`, right?

Reply:
I think this library is what he's talking about: https://www.npmjs.com/package/classnames